src/Wallabag/CoreBundle/Repository/EntryRepository.php

```diff
@@ -17,16 +17,11 @@ class EntryRepository extends EntityRepository
      */
     public function findUnreadByUser($userId)
     {
-        $qb = $this->createQueryBuilder('e')
+        return $this->createQueryBuilder('e')
             ->leftJoin('e.user', 'u')
             ->where('e.isArchived = false')
             ->andWhere('u.id =:userId')->setParameter('userId', $userId)
-            ->orderBy('e.id', 'desc')
-            ->getQuery();
-
-        $pagerAdapter = new DoctrineORMAdapter($qb);
```

Comment:
Sine you removed lines where `DoctrineORMAdapter` is used, you can remove the `use` at the beginning of the file